tethysapp/data_rods_explorer/public/js/cookies.js

```diff
@@ -0,0 +1,37 @@
+var docCookies = {
```

Comment:
Delete this file. I'm pretty sure we replaced the reference to this file with one that comes with Tethys (`{{ tethys.doc_cookies.script_tag|safe }}`).